.eslintrc

```
@@ -0,0 +1,40 @@
+{
+  "extends": ["plugin:prettier/recommended", "prettier", "kentcdodds", "kentcdodds/react", "kentcdodds/jsx-a11y"],
```

```suggestion
  "extends": ["plugin:prettier/recommended", "prettier", "kentcdodds", "kentcdodds/react", "kentcdodds/jsx-a11y", "plugin:@next/next/recommended"],
```

Also use `plugin:@next/next/recommended` otherwise building won't work.